### PR TITLE
Add Dockerfiles for 1.10.3 release

### PR DESCRIPTION
**Pull request by xintongsong**

### CODE REVIEW

File: 1.10/scala_2.11-debian/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# Prepare environment
+ENV FLINK_HOME=/opt/flink
+ENV PATH=$FLINK_HOME/bin:$PATH
+RUN groupadd --system --gid=9999 flink && \
+    useradd --system --home-dir $FLINK_HOME --uid=9999 --gid=flink flink
+WORKDIR $FLINK_HOME
+
+# Install Flink
+RUN set -ex; \
+  wget -O flink.tgz "$FLINK_TGZ_URL"; \
```
Comment (wangyang0918) [inline 1.10/scala_2.11-debian/Dockerfile:61]: It seems that we are using `wget -nv -O flink.tgz "$FLINK_TGZ_URL"` in release 1.10.2.

File: 1.10/scala_2.11-debian/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  \
+  if [ "$CHECK_GPG" = "true" ]; then \
+    wget -O flink.tgz.asc "$FLINK_ASC_URL"; \
```
Comment (wangyang0918) [inline 1.10/scala_2.11-debian/Dockerfile:64]: `wget -nv -O flink.tgz.asc "$FLINK_ASC_URL";`

Same here.

File: 1.10/scala_2.12-debian/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+# Prepare environment
+ENV FLINK_HOME=/opt/flink
+ENV PATH=$FLINK_HOME/bin:$PATH
+RUN groupadd --system --gid=9999 flink && \
+    useradd --system --home-dir $FLINK_HOME --uid=9999 --gid=flink flink
+WORKDIR $FLINK_HOME
+
+# Install Flink
+RUN set -ex; \
+  wget -O flink.tgz "$FLINK_TGZ_URL"; \
```
Comment (wangyang0918) [inline 1.10/scala_2.12-debian/Dockerfile:61]: `wget -nv -O flink.tgz "$FLINK_TGZ_URL";`

Same here.

File: 1.10/scala_2.12-debian/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  \
+  if [ "$CHECK_GPG" = "true" ]; then \
+    wget -O flink.tgz.asc "$FLINK_ASC_URL"; \
```
Comment (wangyang0918) [inline 1.10/scala_2.12-debian/Dockerfile:64]: `wget -nv -O flink.tgz.asc "$FLINK_ASC_URL";`

**Review by wangyang0918 — COMMENTED** [review]
Thanks for working on the release 1.10.3.

This PR looks good to me. I compare the content in this PR with `58a29fca7c6ff05999fad4371638d16335f7e93e`, which introduce the release 1.10.2. And I find some minor differences. It will be great if the comments could be addressed before merging.

**Comment by xintongsong** [comment]
@wangyang0918 ,
Nice catch. This is because I generated this PR based on the [do not merge] commit of #64. I'll fix it.

**Review by wangyang0918 — APPROVED** [review]
+1

**Pull request closed without merging**